### PR TITLE
Polish gh-40023

### PR DESCRIPTION
**Pull request by izeye**
This PR polishes the changes made in gh-40023 a bit.

### CODE REVIEW

File: spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/metrics/export/prometheus/PrometheusSimpleclientMetricsExportAutoConfiguration.java [hunk 1]
```diff
@@ -69,14 +67,14 @@
 		before = { CompositeMeterRegistryAutoConfiguration.class, SimpleMetricsExportAutoConfiguration.class },
 		after = { MetricsAutoConfiguration.class, PrometheusMetricsExportAutoConfiguration.class })
 @ConditionalOnBean(Clock.class)
-@ConditionalOnClass(PrometheusMeterRegistry.class)
+@ConditionalOnClass(io.micrometer.prometheus.PrometheusMeterRegistry.class)
```
Comment (mhalbritter) [inline spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/metrics/export/prometheus/PrometheusSimpleclientMetricsExportAutoConfiguration.java:70]: What's the reason for that change?

**Comment by mhalbritter** [comment]
Hey, thanks for the PR, i've added a comment.

File: spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/metrics/export/prometheus/PrometheusSimpleclientMetricsExportAutoConfiguration.java (hunk 1 above)
Comment (izeye) [reply spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/metrics/export/prometheus/PrometheusSimpleclientMetricsExportAutoConfiguration.java:70]: @mhalbritter I just wanted to align with the fully qualified return type of the `simpleclientPrometheusMeterRegistry()` below and remove the import statement for the `PrometheusMeterRegistry` that has been deprecated.

**Comment by mhalbritter** [comment]
Thanks!

**Pull request closed without merging**